ceph-nvmeof.conf

```diff
@@ -15,6 +15,8 @@ port = 5500
 enable_auth = False
 state_update_notify = True
 state_update_interval_sec = 5
+min_cntlid = 1
+max_cntlid = 65519
```

Comment:
Hello @gbregman 🖖. Why 65519? Would expect 65535 or 0xffff

Reply:
Alex, the current value in SPDK is 0xFFEF, which is 65519. I wanted to keep the current behavior until we set real values in the future using ceph-adm.

Reply:
May be use some more human readable names, like `max/min_controller_id`?
And we seemed to have an agreement to have config params that have default values to be commented out in the sample config.

Reply:
Will rename the fields as suggested.